src/cobertura.rs

```diff
@@ -246,13 +249,12 @@ fn get_coverage(
             }
             start_indexes.sort_unstable();
 
-            let functions = result.functions;
-            let result_lines = result.lines;
-            let result_branches = result.branches;
+            // let result_lines = result.lines.clone();
+            // let result_branches = result.branches.clone();
```

Comment:
Could you revert these changes as they are unrelated?

Reply:
Of course, but why this clone that is not required?

Reply:
You can submit this change as a separate PR (I just want PRs to be self-contained, I'm not opposing to your change)

Reply:
Oh sorry, I didn't realize your change was because of the change to the `results: &[(PathBuf, PathBuf, CovResult)],` argument. You can unrevert it :)

Reply:
dropped the commit